frontend/src/pages/Home.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import StatusBubbles from '../components/StatusBubbles/StatusBubbles';
+import TableComponent from '../components/Table/Table';
+
+// Function to generate column names and data
+const createTableData = (columns, rows) => {
+  return {
+    columns,
+    data: rows
+  };
+};
+
+function Home() {
+  // State to track which table is currently visible
+  const [currentTable, setCurrentTable] = useState('attentionRequired'); // Default to first table
```

Comment:
also, can we extract out the logic of tabs to a separate component that takes children as prop

Reply:
I have extracted the logic of the tabs into a separate component that takes children as a prop. This has been updated in commit [8082c03].